puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/release/katello/nightly.groovy

```diff
@@ -92,6 +92,15 @@ pipeline {
             }
         }
     }
+    post {
+        failure {
+            emailext(
+                subject: "${env.JOB_NAME} failed",
+                to: 'ci@community.theforeman.org',
+                body: "Foreman nightly pipeline failed: \n\n${env.BUILD_URL}"
```

Comment:
Should this string be unique from the one above, since it's from a different build?

Reply:
Doh, good catch :)